views/forms/TextInputTopLabel/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { InputLabel, FormHelperText, OutlinedInput } from '@mui/material/'
+import { useController } from 'react-hook-form'
+
+import { lineHeight, fontSize, borderRadius } from '../../../constants'
+
+const TextInputTopLabel = (props) => {
+  const {
+    field,
+    formState: { errors },
+  } = useController(props)
+
+  return (
+    <>
+      <InputLabel
+        {...props?.inputLabel}
+        aria-label={props.label}
+        shrink
+        sx={{
+          color: 'text.primary',
+          fontWeight: 500,
+          lineHeight: lineHeight[21],
+          fontSize: fontSize[18],
+          ...props.inputLabel?.sx,
+        }}
+        required={props.required}
+        htmlFor={props.name}
+      >
+        {props.label}
+      </InputLabel>
+      <OutlinedInput
+        {...field}
+        aria-invalid={!!errors ? 'true' : 'false'}
```

Comment:
I think this should look for errors on the field instead of any errors.

Reply:
[a8fa89f](https://github.com/AMP-SCZ/dpdash/pull/478/commits/a8fa89f3d10b6e4db122a9aa52573c428d3682dc)